app/views/guests/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Guest Info Page</h1>
+
+<p>Guest Name: <%= @guest.name %></p>
+
+<ul>
+  <% @guest.rooms.each do |room| %>
+  <p>Suite: <%= room.suite %></p>
+  <p>Cost: $<%= room.rate %></p>
+  <p>Hotel Name: <%= room.hotel.name %></p>
+  <% end %>
+</ul>
+
+<%= form_with url: "/guest_rooms?guest_id=#{@guest.id}", method: :post, local:true do |form| %>
+  <%= form.text_field :room_id %>
+  <%= form.submit "Add Room" %>
+<% end %>
```

Comment:
The idea behind this form is spot on, you need to get access to the guest ID and the room ID to make that connection in the controller, but like I mentioned in the `routes.rb` file, the path is not quite correct. With a correct path of `"/guests/#{@guest.id}/rooms"`, we'll get access to the guest ID from the route, and the room ID from the form body.